.travis.yml

```diff
@@ -44,7 +44,7 @@ matrix:
             - ubuntu-toolchain-r-test
           packages:
             - g++-5
-    - env: CXX=clang++-4.0 CC=clang-4.0
+    - env: CXX=clang++-4.0 CC=clang-4.0 CXXFLAGS=-stdlib=libc++
```

Comment:
This shouldn't be needed?

https://github.com/gerbera/gerbera/blob/58b7f800f3b82e883257a4464e735564f9611aab/CMakeLists.txt#L286

Reply:
Golly, I flew by that statement....For simplicity, I will roll the retrieve back to a standard `find_package(...)`  That will eliminate some complexities and solve the offline scenario.